lib/modules/datasource/sbt-plugin/index.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('modules/datasource/sbt-plugin/index', () => {
         .get('/maven/org/scalatest/')
         .reply(404);
       httpMock
-        .scope('https://repo.maven.apache.org')
+        .scope('https://repo1.maven.org')
```

Comment:
> I've started a discussion here with more detail about what seems to be going wrong: #29755
> I'm not sure this pull request will fix that problem but this pull request at least gives better defaults (new repo first, outdated repo second).

I'm marking this PR as `status:blocked`. Once we are ready to merge, a maintainer should:

- [ ] Remove the `status:blocked` label
- [ ] Resolve this review comment